csharp/Perf/Microsoft.Spark.CSharp/Program.cs

```diff
@@ -24,6 +25,15 @@ class PerfBenchmark
 
         public static void Main(string[] args)
         {
+            if (args.Length != 3)
+            {
+                var exe = System.Reflection.Assembly.GetEntryAssembly().Location;
+                Console.WriteLine(@"Usage   : {0}  spark-local-dir   run-count  data-path", exe);
+                Console.WriteLine(@"Example : {0}  D:\Temp\perfTest  10         hdfs:///perfdata/freebasedeletions/*", exe);
```

Comment:
Yes, such as `file:///d:/mobius/deletions/deletions.csv-00000-of-00020`.
I just give 2 cluster examples of hdfs file.
Do you want me to add a case of this format data path?